erpnext/public/js/controllers/transaction.js

```diff
@@ -209,6 +209,17 @@ erpnext.TransactionController = erpnext.taxes_and_totals.extend({
 			});
 		}
 
+		if (this.frm.fields_dict.taxes_and_charges) {
+			this.frm.set_query("taxes_and_charges", function() {
```

Comment:
`taxes_and_charges` field is there in purchase cycle also, but it is linked to `Purchase Taxes and Charges Template`.

After this change, seems like purchase cycle doctype will also be linked to "Sales Taxes and Charges Template".

Reply:
Fixed:
Removed the doctype name since it gets added according to the doctype in link.js